projects/Mallard/src/App.tsx

```diff
@@ -161,7 +161,7 @@ export default class App extends React.Component<{}, {}> {
                         <View style={styles.appContainer}>
                             <RootNavigator
                                 {...rootNavigationProps}
-                                enableURLHandling={false}
```

Comment:
This changes brings back nav persistence in dev (we don't want URL handling in PROD)